src/ConfigDiscoveryTrait.php

```diff
@@ -118,7 +116,7 @@ private function getApplicationConfig()
             return $this->applicationConfig;
         }
 
-        $configFile = isset($this->projectDir)
```

Comment:
This change seems wrong, considering the scope of the trait

Reply:
Ok, rolled back to isset with update psalm config https://github.com/laminas/laminas-development-mode/pull/23/commits/b592b61bed6e5ac4e8faf85ce6dde1b8856b9cce